safe_transaction_service/history/indexers/internal_tx_indexer.py

```diff
@@ -178,7 +184,7 @@ def _find_relevant_elements_using_trace_filter(
                 to_traces = self.ethereum_client.tracing.trace_filter(
                     from_block=from_block_number,
                     to_block=to_block_number,
-                    to_address=addresses,
+                    to_address=list(addresses),
```

Comment:
Finally we use a list here :)

Reply:
`set` should work, but better to use a `list` as it's the expected parameter